html/feed/feed.js

```diff
@@ -1,5 +1,369 @@
+        if (response && response.ok){
+          card.remove(); 
+        }
+      });
+
+    }
+      cardBody.appendChild (userNameAndTimeDiv); 
+
+
+// post Title 
+
+const pTitle = document.createElement ('p'); 
+pTitle.className ='d-flex justify-content-between'; 
+
+const spanTitle= document.createElement ('span'); 
+spanTitle.textContent = element.title; 
+pTitle.appendChild (spanTitle);
+cardBody.appendChild (pTitle); 
+
+//post Body 
+
+const p = document.createElement ('p'); 
+p.className= 'card-text'; 
+p.textContent = element.body; 
+
+if (element.media) {
+
+  const mediaWrapper = document.createElement ('div'); 
+  mediaWrapper.className = 'w-100'; 
+
+  const postMedia = document.createElement ('img'); 
+  postMedia.src = element.media; 
+  postMedia.className = 'rounded img-fluid'; 
+
+  mediaWrapper.appendChild (postMedia); 
+  cardBody.appendChild (mediaWrapper); 
+
+}
+
+cardBody.append (p)
+
+
+// more Btns 
+
+const btnGroup = document.createElement ('div'); 
+btnGroup.className ='d-flex'; 
+
+const btnComment = document.createElement ('button'); 
+btnComment.className = 'btn btn-link btn-sm text-muted'; 
+
+const iComment = document.createElement ('i'); 
+iComment.className = 'fas fa-comment'; 
+btnComment.appendChild (iComment);
+
+const spanComment = document.createElement ('span'); 
+spanComment.id= 'commentCount'; 
+spanComment.textContent = element._count.comments; 
+btnComment.appendChild (spanComment); 
+
+const btnLike = document.createElement ('button'); 
+btnLike.className = 'btn btn-link btn-sm text-muted'; 
+
+const iLike = document.createElement ('i'); 
+iLike.className = 'fas fa-heart text-danger'; 
+btnLike.appendChild (iLike); 
+
+const spanLike = document.createElement ('span'); 
+spanLike.id= `likeCount${element.id}`; 
+spanLike.textContent = element.reactions [0]?.count; 
+
+console.log (element)
+btnLike.appendChild (spanLike); 
+
+// edit btn 
+
+if (userName === element.author.name){
+
+  const btnEdit = document.createElement ('button'); 
+  btnEdit.className = 'btn btn-link btn-sm text-muted'; 
+  
+  const iEdit = document.createElement ('i'); 
+  iEdit.className = 'fas fa-edit'; 
+  btnEdit.appendChild (iEdit); 
+
+  const spanEdit = document.createElement ('span'); 
+  spanEdit.textContent = "Edit"; 
+  btnEdit.setAttribute ('data-post-id', element.id); 
+  confirmEditBtn.setAttribute ('data-post-id', element.id); 
+  btnEdit.appendChild (spanEdit)
+
+  //add eventListener to edit post
+
+  btnEdit.addEventListener ('click',function (){
+
+    document.getElementById ('editTitle').value = element.title; 
+    document.getElementById ('editBody').value = element.body; 
+
+    var modal =  new bootstrap.Modal (getElementById ('editPostModal')); 
+
+    modal.show ();
+  }); 
+
+  btnGroup.appendChild (btnEdit);
+}
+
+// likeBtn eventListener 
+
+btnLike.addEventListener ("click", (e)=>{
+  reactToPosts (baseUrl, element.id, token, spanlike)
```

Comment:
spanlike is typed wrong which causes an error in the console.

Reply:
changed it-  spanLike